Assets/GothicVR/Scripts/UI/DropdownController.cs

```diff
@@ -0,0 +1,52 @@
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.XR.Interaction.Toolkit;
+using GVR.Phoenix.Interface;
+public class DropdownController : MonoBehaviour
```

Comment:
This controller is handling Snap-turn changes only. Please rename it to reflect this. e.g. _SnapTurnSettingsController_